.travis.yml

```diff
@@ -11,11 +11,15 @@ before_install:
     - sudo apt update
     - sudo apt install -y quantum-espresso 
     - pip install codecov pytest pytest-cov pytest_mock
+    - git clone https://github.com/lammps/lammps.git lammps
```

Comment:
consider e.g. `--depth-1` for faster download (although I'm guessing the internet speed is pretty good in the cloud)